d3a_interface/device_validator.py

```diff
@@ -42,26 +45,29 @@ def validate_load_device(**kwargs):
             {"misconfiguration": [f"daily_load_profile shouldn't be set with "
                                   f"avg_power_W, hrs_per_day & hrs_of_day."]})
     if ("hrs_per_day" in kwargs and kwargs["hrs_per_day"] is not None) and \
-            not (LoadSettings.HOURS_RANGE.min <= kwargs["hrs_per_day"] <=
-                 LoadSettings.HOURS_RANGE.max):
+            not _validate_range_limit(LoadSettings.HOURS_RANGE.initial,
+                                      kwargs["hrs_per_day"],
+                                      LoadSettings.HOURS_RANGE.final):
         raise D3ADeviceException(
             {"misconfiguration": [f"hrs_per_day should be in between "
-                                  f"{LoadSettings.HOURS_RANGE.min} & "
-                                  f"{LoadSettings.HOURS_RANGE.max}."]})
+                                  f"{LoadSettings.HOURS_RANGE.initial} & "
```

Comment:
Sorry, maybe I wasn't as explicit as i should have been. IMO For the HOURS_RANGE initial and final doesnt make sense. 
I thought you ad another Rangelimit for the rates and keep this one for the non-rate values. Does it make sense?

Reply:
Im not sure if its really that important, at least from the wordings of the ticket `Please change the naming of SELLING_RANGE and BUYING_RANGE constants and also the naming of the NamedTuple RangeLimit  from min/max to initial/final`. I had an impression to simply change min/max to initial/final. 
If its important to classify in two different namedtuples then please also tell me where it need to (min, max) namedtuple and where it should be (initial, final)